.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: go
+
+go:
+  - 1.3
+  - 1.4
+  - tip
+
+before_install:
+  # install linting tools
+  - go get github.com/golang/lint/golint
+  - go get golang.org/x/tools/cmd/vet
+  - go get github.com/fzipp/gocyclo
+  # install code coverage tools
+  - go get github.com/axw/gocov/gocov
+  - go get github.com/mattn/goveralls
+  - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
+
+install:
+  - cd lib/goose
+  - go get -t
+  - go build
+  - cd ../..
```

Comment:
While I think the long term solution is to fix it so that `go build ./...` doesn't break due to the `db-sample/migrations` directory, can we at least pretty this up to remove the `cd`:

    go get -t ./lib/goose
    go build ./lib/goose
    go get -t ./cmd/goose
    go build ./cmd/goose